server/database/models/token.js

```diff
@@ -0,0 +1,19 @@
+"use strict";
+module.exports = (sequelize, DataTypes) => {
+  const Token = sequelize.define(
+    "Token",
+    {
+      access_token: DataTypes.STRING,
+      refresh_token: DataTypes.STRING,
+      scope: DataTypes.STRING,
+      token_type: DataTypes.STRING,
+      expiry_date: DataTypes.BIGINT,
+      userId: DataTypes.INTEGER,
```

Comment:
can you make this a primary key actually? It'll make it more efficient to get, and it actually is a primary key